exercise/templates/exercise/_user_results.html

```diff
@@ -146,14 +146,14 @@ <h3 class="panel-title">
         {% if is_course_staff %}
         <td>
           {% if not accessible %}
-          <a class="btn btn-default btn-xs" href="{{ entry.link }}">
+          <a class="aplus-button--secondary aplus-button--xs" role="button" href="{{ entry.link }}">
```

Comment:
Now that you removed button classes from "early access" links, this one was probably an oversight since it still has button classes.

Reply:
Thanks, I fixed it now. It seems to be the ones where there are multiple in one file that I'm liable to miss!